orchestrator/dockerdir/etc/orchestrator/orchestrator.conf.json

```diff
@@ -0,0 +1,50 @@
+{
+  "ApplyMySQLPromotionAfterMasterFailover": false,
+  "BackendDB": "sqlite",
+  "Debug": true,
+  "DetachLostReplicasAfterMasterFailover": true,
+  "DetectClusterAliasQuery": "SELECT CONCAT(SUBSTRING(@@hostname, 1, LENGTH(@@hostname) - 1 - LENGTH(SUBSTRING_INDEX(@@hostname,'-',-2))),'.',SUBSTRING_INDEX(@@report_host,'.',-1))",
+  "DetectInstanceAliasQuery": "SELECT @@hostname",
+  "DiscoverByShowSlaveHosts": false,
```

Comment:
This should be true as we discussed earlier.

Reply:
yes, I remember it don't worry